### ISSUE TITLE
Seems like google is refreshing my access token every few minutes 😥

### ISSUE THREAD

**Issue by chagai95** [costastf/locationsharinglib#114]
Did anyone else experience this? Is there anything I can do?

I have it behind this API:
https://api.chagai.website/location

**Comment by costastf** [comment]
Hi, first I would ask how are you getting your cookies. If you are using an account that you are actually using then you might be getting new cookies often. It is suggested to use a third account that is not in everyday use and share your location to that one. When you export the cookies use a cookie deleting add on so the session does not terminate if you close your browser. That's what I can think of right now.

**Comment by chagai95** [comment]
Thx for your feedback, I guess it's time for a new account, I was using one especially for this - whereischagai 😂 but I guess I used it too often...

**Comment by costastf** [comment]
Good luck with a new account. Please feel free to close this if you think that your question is answered.

**Comment by chagai95** [comment]
Yep it works great! Just wanted to wait a few hours to see if it still works 😎

**Comment by chagai95** [comment]
Hey, I need to reopen this, I just opened a new account and the cookies are invalidated again... I'm going to open another new account just to make sure this is actually happening consistently, or maybe it's related to me having shared my location with this account?

**Comment by chagai95** [comment]
it's been working for an hour, I'll keep this updated

**Comment by chagai95** [comment]
It's still working so I'm gonna close this, I think it might be related to not having verified some security measure before sharing the location with my main account.

**Comment by chagai95** [comment]
I am having this issue again so I'm going to re-open... I tried creating two new accounts already and it still invalidates the caches after a few minutes/hours....

**Comment by chagai95** [comment]
Hey everyone, I'm back here after nearly a year - unfortunately still facing the same issue, anyone else in the same boat as me?

**Comment by chagai95** [comment]
It's working now but it'll stop in a few minutes I fear..

https://api.chagai.website:4400/location

**Comment by chagai95** [comment]
It stopped working but I just grabbed a new cookie a few days ago and it's still working! Let's see how long until it stops working again...
